sky/serve/autoscalers.py

```diff
@@ -67,16 +67,24 @@ def __init__(self, spec: 'service_spec.SkyServiceSpec') -> None:
         """
         self.min_replicas: int = spec.min_replicas
         self.max_replicas: int = spec.max_replicas or spec.min_replicas
+        # Target number of replicas is initialized to min replicas.
+        # TODO(MaoZiming): add init replica numbers in SkyServe spec.
         self.target_num_replicas: int = spec.min_replicas
 
+    def update_spec(self, version: int, spec: 'service_spec.SkyServiceSpec',
+                    mixed_replica_versions: bool) -> None:
+        del version, mixed_replica_versions  # Unused.
+        self.min_nodes = spec.min_replicas
+        self.max_nodes = spec.max_replicas or spec.min_replicas
+        self.target_num_replicas = spec.min_replicas
+
     def collect_request_information(
             self, request_aggregator_info: Dict[str, Any]) -> None:
         """Collect request information from aggregator for autoscaling."""
         raise NotImplementedError
 
     def evaluate_scaling(
-        self,
-        replica_infos: List['replica_managers.ReplicaInfo'],
+        self, replica_infos: List['replica_managers.ReplicaInfo']
```

Comment:
```suggestion
        self, replica_infos: List['replica_managers.ReplicaInfo'],
```
By adding the comma at the final arguments it will format as multiple line